code/modules/background/defines.dm

```diff
@@ -18,6 +18,8 @@
 
 #define CITIZENSHIP_NONE "None"
 
+var/global/list/citizenship_list = list(CITIZENSHIP_BIESEL, CITIZENSHIP_SOL, CITIZENSHIP_COALITION, CITIZENSHIP_ELYRA, CITIZENSHIP_ERIDANI, CITIZENSHIP_DOMINIA, CITIZENSHIP_IZWESKI, CITIZENSHIP_JARGON, CITIZENSHIP_PRA, CITIZENSHIP_DPRA, CITIZENSHIP_NKA, CITIZENSHIP_ZORA, CITIZENSHIP_NONE)
```

Comment:
Kinda pointless to have list, you could just the species list instead or the list that exists already in SSrecords.